akka-http-core/src/main/mima-filters/10.0.9.backwards.excludes

```diff
@@ -0,0 +1,7 @@
+# Changes allow for adding proxy auth headers to the client transport.
+# Changes allow for backwards compatibility when auth is not needed. Most changes are to internal classes only.
+ProblemFilters.exclude[MissingTypesProblem]("akka.http.scaladsl.ClientTransport$HttpsProxyTransport$")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.scaladsl.ClientTransport#HttpsProxyTransport.apply")
+ProblemFilters.exclude[DirectMissingMethodProblem]("akka.http.scaladsl.ClientTransport#HttpsProxyTransport.copy")
```

Comment:
(That would have to be implemented on the Migration Manager, the tool that checks bin compat)